arangod/Pregel/Algos/EffectiveCloseness/EffectiveCloseness.cpp

```diff
@@ -94,15 +94,14 @@ struct ECGraphFormat : public GraphFormat<ECValue, int8_t> {
                          std::string const& result)
       : GraphFormat<ECValue, int8_t>(server), _resultField(result) {}
 
-  size_t estimatedEdgeSize() const override { return 0; };
+  size_t estimatedEdgeSize() const override { return 0; }
 
-  void copyVertexData(std::string const& documentId, arangodb::velocypack::Slice document,
-                      ECValue& targetPtr) override {}
+  void copyVertexData(std::string const& /*documentId*/, arangodb::velocypack::Slice /*document*/,
+                      ECValue& /*targetPtr*/, uint64_t& /*vertexIdRange*/) override {}
 
-  void copyEdgeData(arangodb::velocypack::Slice document, int8_t& targetPtr) override {}
+  void copyEdgeData(arangodb::velocypack::Slice /*document*/, int8_t& /*targetPtr*/) override {}
```

Comment:
One could add a empty default implementation in the base class. Its almost always empty.

Reply:
Ok, I will adjust the default implementation of `copyEdgeData` to do nothing, and remove the empty overrides for it.
`buildEdgeDocument` I already removed in this PR, because I could not find any override of it that would do anything.